### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [openopps/openopps-platform#1257]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)
## Results for https://pages.18f.gov/openopps-platform/
- **Error** The html element should have a lang or xml:lang attribute which describes the language of the document.
  - WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
  - `html`
  - `<html><head>
    <meta charset="utf-8">...</html>`
- **Error** This link points to a named anchor "how-you-can-help" within the document, but no anchor exists with that name.
  - WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchID
  - `html > body > a`
  - `<a href="#how-you-can-help"><img style="position: absolute;...</a>`
### Summary

**Errors:** 2

**Comment by LisaLNelson** [comment]
What std are we mandated to follow?

**Comment by kbwagner** [comment]
Closing per backlog review.
